delivery_auto_refresh/models/sale_order.py

```diff
@@ -122,7 +122,7 @@ def _is_delivery_line_voidable(self):
         # were not set to refund.
         qty_delivered = sum(
             self.order_line.filtered(
-                lambda x: not x.is_delivery and x.product_id.detailed_type != "service"
+                lambda x: not x.is_delivery and x.product_id.is_storable
```

Comment:
Consumable products also count:

```suggestion
                lambda x: not x.is_delivery and x.product_id.type == "product"
```